tests/monitoring_test.go

```diff
@@ -154,6 +156,41 @@ var _ = Describe("Prometheus Alerts", func() {
 			waitForAlertToActivate("SSPDown")
 		})
 	})
+
+	Context("DeprecatedRHEL6Vm Alert", func() {
+		var (
+			vm  *kubevirtv1.VirtualMachine
+			vmi *kubevirtv1.VirtualMachineInstance
+		)
+
+		BeforeEach(func() {
+			vmi = NewRandomVMIWithBridgeInterface(strategy.GetNamespace())
+			vm = NewVirtualMachine(vmi)
+			vm.ObjectMeta.Labels = map[string]string{
+				"vm.kubevirt.io/template": "rhel6-desktop-large",
+			}
+			vm.Spec.Running = pointer.Bool(true)
+			eventuallyCreateVm(vm)
+		})
+
+		AfterEach(func() {
+			Expect(apiClient.Delete(ctx, vm)).ToNot(HaveOccurred(), "Failed to delete vm: %s", vm.Name)
+		})
+
+		It("Should fire the DeprecatedRHEL6Vm alert if there is a rhel6 running vm", func() {
+			waitForAlertToActivate(metrics.Rhel6AlertName)
+		})
+
+		It("Should deactivate the DeprecatedRHEL6Vm alert if the rhel6 running vm is stopped", func() {
+			waitForAlertToActivate(metrics.Rhel6AlertName)
+			foundVm := &kubevirtv1.VirtualMachine{}
+			err := apiClient.Get(ctx, client.ObjectKeyFromObject(vm), foundVm)
+			Expect(err).ToNot(HaveOccurred())
+			foundVm.Spec.Running = pointer.Bool(false)
+			Expect(apiClient.Update(ctx, foundVm)).ToNot(HaveOccurred())
```

Comment:
These lines should be wrapped in an `Eventually()` block, because the update may fail when VM status is updated concurrently.

Reply:
I was thinking to use `Patch` instead of `Update`, avoiding the `Eventually` block. WDYT?

Reply:
Ok, you can use `Patch`, but it will probably be more code than using `Eventually()`.

Reply:
At the end I prefer to use your suggestion :)